skimage/restoration/j_invariant.py

```diff
@@ -258,15 +258,14 @@ def calibrate_denoiser(image, denoise_function, denoise_parameters, *,
 
     Examples
     --------
-
     >>> from skimage import color, data
-    >>> from skimage.restoration import denoise_wavelet
+    >>> from skimage.restoration import denoise_tv_chambolle
     >>> import numpy as np
     >>> img = color.rgb2gray(data.astronaut()[:50, :50])
     >>> rng = np.random.default_rng()
     >>> noisy = img + 0.5 * img.std() * rng.standard_normal(img.shape)
-    >>> parameters = {'sigma': np.arange(0.1, 0.4, 0.02)}
-    >>> denoising_function = calibrate_denoiser(noisy, denoise_wavelet,
+    >>> parameters = {'weight': np.arange(0.01, 0.3, 0.02)}
+    >>> denoising_function = calibrate_denoiser(noisy, denoise_tv_chambolle,
     ...                                         denoise_parameters=parameters)
```

Comment:
Not sure if replacing `denoise_wavelet` with `denoise_tv_chambolle` here (and elsewhere) makes sense but it prevents having to skip these doctests. Found an example of `denoise_tv_chambolle` being used with `calibrate_denoiser` in [this gallery example](https://scikit-image.org/docs/stable/auto_examples/filters/plot_j_invariant_tutorial.html#comparing-different-classes-of-denoiser).